src-java/northbound-service/northbound-api/src/main/java/org/openkilda/northbound/dto/v2/yflows/YFlowPatchPayload.java

```diff
@@ -46,6 +46,7 @@
     Boolean strictBandwidth;
     String description;
     Boolean allocateProtectedPath;
+    String diverseFlowId;
```

Comment:
Is it a flow or y-flow? Shouldn't be diverseYFlowId?

Reply:
As we discussed offline, this can be either flow or y-flow.